public/css/styles.css

```
@@ -3,9 +3,11 @@
    ================================ */
 
 /* Import Google Fonts */
+
 @import url('https://fonts.googleapis.com/css2?family=Inter:wght@300;400;500;600;700;800&display=swap');
```

Stylelint (17.12.0)

[error] Expected "url('https://fonts.googleapis.com/css2?family=Inter:wght@300;400;500;600;700;800&display=swap')" to be "'https://fonts.googleapis.com/css2?family=Inter:wght@300;400;500;600;700;800&display=swap'" (import-notation)

(import-notation)